agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/httpclient/RedirectPolicy.java

```diff
@@ -60,8 +64,14 @@
       HttpRequest originalHttpRequest,
       int retryCount) {
     // make sure the context is not modified during retry, except for the URL
+    String redirectLocation = null;
```

Comment:
see https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#final-keyword-usage

> Local variables should only be declared `final` if they are not initialized inline
(declaring these vars `final` can help prevent accidental double-initialization).

```suggestion
    final String redirectLocation;
```